src/lib/deploy.js

```diff
@@ -344,10 +344,22 @@ export async function deploy({ alias, yes }) {
   let transactionJson = transaction.json;
   let { feepayerAlias, url } = config.deployAliases[alias];
   const settings = [
-    [chalk.bold('Deploy Alias'), chalk.reset(alias)],
-    [chalk.bold('Fee Payer Alias'), chalk.reset(feepayerAlias)],
+    [chalk.bold('Deploy alias'), chalk.reset(alias)],
+    [chalk.bold('Network kind'), chalk.reset(networkId)],
     [chalk.bold('URL'), chalk.reset(url)],
-    [chalk.bold('Smart Contract'), chalk.reset(contractName)],
+    [
+      chalk.bold('Fee payer'),
+      chalk.reset(
+        `Alias         : ${feepayerAlias}\nAccount       : ${feepayerAddressBase58}`
+      ),
+    ],
+    [
+      chalk.bold('zkApp'),
+      chalk.reset(
+        `Smart contract: ${contractName}\nAccount       : ${zkAppAddress.toBase58()}`
```

Comment:
Good question for separate issue. This one is simple improvement that we can deliver right away but your question is more complicated and needs more time (discussion, design, implementation, testing).

Reply:
@MartinMinkov I think it is a good idea to list all the dependancies somewhere, but as @shimkiv mentioned it will necessitate more discussion and thought to do it well. We will have to think about how to display the dependancies in a intuitive way, wether or not we should store the dependency tree in the config and what the best data structure to do so etc etc. All that said, I like the idea.